kotlin/workflow-host/src/main/java/com/squareup/workflow/internal/RealWorkflowContext.kt

```diff
@@ -81,7 +81,7 @@ internal class RealWorkflowContext<StateT : Any, OutputT : Any>(
         key: String,
         handler: (ChildOutputT) -> WorkflowAction<StateT, OutputT>
       ): ChildRenderingT {
-  // @formatter:on
+    // @formatter:on
```

Comment:
til

Reply:
Support for this syntax is not on by default in IntelliJ unfortunately, but it'll break ktlint if you accidentally reformat without it.